<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Nexgen84" failures="1" tests="15" name="com.ots.test.ui.NegativeLoginTest" time="137.236" errors="1" timestamp="2022-11-03T21:50:41 IST" skipped="6">
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="login" time="15.031"/>
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="otsDateTestCheckEmptyDate" time="40.074"/>
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="otsDateEmptyDateFieldAndErrorMessage" time="6.141">
    <failure message="expected:&lt;[Date must be future date or greater than previous date]&gt; but was:&lt;[Fields can&#039;t be empty]&gt;" type="org.junit.ComparisonFailure">
      <![CDATA[org.junit.ComparisonFailure: expected:<[Date must be future date or greater than previous date]> but was:<[Fields can't be empty]>
at org.junit.Assert.assertEquals(Assert.java:117)
at org.junit.Assert.assertEquals(Assert.java:146)
at com.ots.test.ui.NegativeLoginTest.otsDateEmptyDateFieldAndErrorMessage(NegativeLoginTest.java:165)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- otsDateEmptyDateFieldAndErrorMessage -->
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="runReportWithEmptyDate" time="15.831"/>
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="otsDatePastDateTest" time="12.457"/>
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="otsDatePastDateErrorText" time="6.110"/>
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="otsDatefutureDate" time="23.325"/>
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="otsDateAddDateFieldFutureDayTest" time="15.228"/>
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="addedDateFieldTest" time="3.039">
    <error message="Index 1 out of bounds for length 1" type="java.lang.IndexOutOfBoundsException">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 1 out of bounds for length 1
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at com.ots.test.ui.NegativeLoginTest.addedDateFieldTest(NegativeLoginTest.java:308)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- addedDateFieldTest -->
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="otsDatePresentDate" time="0.000">
    <skipped/>
  </testcase> <!-- otsDatePresentDate -->
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="runReportWithSameDate" time="0.000">
    <skipped/>
  </testcase> <!-- runReportWithSameDate -->
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="runReportWithPastDate" time="0.000">
    <skipped/>
  </testcase> <!-- runReportWithPastDate -->
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="runReportWithPastDateField" time="0.000">
    <skipped/>
  </testcase> <!-- runReportWithPastDateField -->
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="runReportDatePresentDate" time="0.000">
    <skipped/>
  </testcase> <!-- runReportDatePresentDate -->
  <testcase classname="com.ots.test.ui.NegativeLoginTest" name="runReportDateNoOption" time="0.000">
    <skipped/>
  </testcase> <!-- runReportDateNoOption -->
</testsuite> <!-- com.ots.test.ui.NegativeLoginTest -->
